bulkExport/glueScripts/export-script.py

```diff
@@ -60,12 +63,21 @@
     }
 )
 
+print('Start filtering by tenantId')
+# Filter by resource listed in Type and with correct STATUS
+if (tenantId is None):
+    filtered_tenant_id_frame = original_data_source_dyn_frame
+else:
+    filtered_tenant_id_frame = Filter.apply(frame = original_data_source_dyn_frame,
+                               f = lambda x:
+                               x['_tenantId'] == tenantId)
```

Comment:
instead of doing glue side filtering would it be better to have a secondary index on the tenantId? This will become an expensive operation if we have to scan across all tenants

Reply:
Emm that's a great question. In the design doc, it specified the filtering is to be done as part of the Glue job, and secondary index was not introduced for any tables. @carvantes Any thoughts?

Reply:
The glue job always scans the entire DDB table no matter what, there's no way to use a query. This is a limitation on the current AWS Glue + DDB integration.

There are existing scenarios where this is far from ideal. e.g. exporting a single FHIR resource type or exporting the resources modified in the last hour will both scan the entire table.

There is room for improvement on the bulk export solution, but we are not changing the fundamentals here.